docs/usage.md

```diff
@@ -69,119 +115,360 @@ cat ./compilation.json
+CodeChecker fixit --checker-name readability-else-after-return \
+   --apply  ./reports
+CodeChecker check --ignore skip.list --output ./reports --enable-all \
+    --logfile ./compile_commands.json
 ```
 For more details regarding the skip file format see
 the [user guide](analyzer/user_guide.md#skip).
 
-c) **Select source files from the compilation database**
+#### Analyze explicitly selected source files from the compilation database<a name="analize-explicit-files"></a>
 You can select which files you would like to analyze from the compilation
 database. This is similar to the skip list feature but can be easier to quickly
 analyze only a few files not the whole compilation database.
 
-Let's assume you have the compilation database in `compile_commands.json`.
+Undo filesystem modifications as described in the
+[previous section](#narrow-files). Re-do analyze and perform automatic fix. The
+following command re-analyzes only the `main.cpp` file.
 
 ```sh
-CodeChecker check ./compile_commands.json -o reports --file "*cmd-find.c"
+CodeChecker check --output ./reports --file "*/src/main.c" \
+    --enable-all --logfile ./compile_commands.json
 ```
 
 Absolute directory paths should start with `/`, relative directory paths should
 start with `*` and it can contain path glob pattern. Example:
 `/path/to/main.cpp`, `lib/*.cpp`, `*/test*`.
 
+## Step 5: Cross Translation Unit analization<a name="step-5"></a>
+The previous analization sessions did not follow dependencies between translation units. Make a try with CTU analysis. The `--ctu` option should be
+added to analyze command. Choose an other "report-directory", for example
+`./reports-ctu` to be able to compare output of different analysis configs.
+
+```sh
+CodeChecker analyze ./compile_commands.json --output ./reports-ctu \
+    --enable-all --ctu
+```
+
+In this case the analyzer configuration enabled and we expect that the cross
+translation unit checking found more issues.
+
+```sh
+CodeChecker parse --print-steps ./reports-ctu
+```
+
+The example code have an other bug!
+
+```sh
+[HIGH] .../docs/examples/src/divide.c:5:22: Division by zero [core.DivideZero]
+    return numerator / denominator;
+                     ^
+  Report hash: 0865bada8cc2c49d99f981d8c6484bd7
+  Steps:
+    1, main.c:17:44: Passing the value 0 via 2nd parameter 'denominator'
+    2, main.c:17:26: Calling 'divide'
+    3, divide.c:3:1: Entered call from 'main'
+    4, divide.c:5:22: Division by zero
+
+Found 1 defect(s) in divide.c
+
+```
+
+## Step 6: Store analysis results in a CodeChecker DB and visualize results<a name="step-6"></a>
+You can store the analysis results in a central database and view the results
+in a web viewer:
+
+1. Start the CodeChecker server locally on port 8555 (using SQLite DB, which is
+not recommended for multi-user central deployment) create a workspace
+directory, where the database will be stored.
+
+Note: Use a new shell.
+
+```sh
+cd <repo root dir>
+mkdir ./ws
```

Comment:
You don't have to create the workspace directory, CodeChecker will create it:
```suggestion
```

Reply:
Removed.